tests/unit/test_media_backfill_unit.py

```
@@ -0,0 +1,168 @@
+        fixed_now = dt.datetime(2026, 1, 1, 2, 0, 0, tzinfo=dt.UTC)
+        fake_datetime = types.SimpleNamespace(now=Mock(return_value=fixed_now))
+        with patch.object(self.module, "datetime", fake_datetime):
+            args = argparse.Namespace(window_start=None, window_end=None, since_minutes=30)
+            start, end = self.module.resolve_window(args)
+
+        self.assertEqual(end, fixed_now)
+        self.assertEqual(start, fixed_now - dt.timedelta(minutes=30))
+
+    def test_resolve_window_requires_arguments(self):
+        args = argparse.Namespace(window_start=None, window_end=None, since_minutes=None)
+        with self.assertRaises(ValueError):
```

Ruff (0.15.9)

[warning] Use `pytest.raises` instead of unittest-style `assertRaises`

Replace `assertRaises` with `pytest.raises`

(PT027)